bundles/binding/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/common/Pck.java

```diff
@@ -0,0 +1,584 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lcn.common;
+
+/**
+ * Helpers to generate and parse LCN-PCK commands.
+ * <p>
+ * LCN-PCK is the command-syntax used by LCN-PCHK to send and receive LCN commands.
+ * 
+ * @author Tobias J�ttner
+ */
+public final class Pck {
```

Comment:
i would suggest to rename this class into (at least) PckParser since this seem to be purpose of this class (parsing)